File: training task/encapsulation_public.py
class StudentPublic:
    def __init__(self, name, age,address):
        self.name = name  
        self.age = age   
        self.address = address   

    def display_details(self):
        print(f"Name: {self.name}, Age: {self.age}, Address: {self.address}")

student = StudentPublic("Nitu", 20,"Moulovipara")

print("Original Details:")
student.display_details()

student.name = "Mou"
student.age = 22
student.address = "sonadanga"

print("Modified Details:")
student.display_details()

File: training task/encapsulation_public.cpp
#include <iostream>
#include <string>

using namespace std;

class StudentPublic {
public:
    string name;
    int age;
    StudentPublic(string name, int age) {
        this->name = name;
        this->age = age;
    }
   void displayDetails();
    
    
};
void StudentPublic:: displayDetails(){
        cout << "Name: " << name << ", Age: " << age << endl;
    }

int main() {
    StudentPublic student("Mohua", 20);
    cout << "Original Details:" << endl;
    student.displayDetails();

    student.name = "Nasima";
    student.age = 22;

    cout << "Modified Details:" << endl;
    student.displayDetails();

    return 0;
}
